src/blocks/buttons/settings.js

```diff
@@ -1,6 +1,6 @@
 import { memo } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
-import { InspectorControls } from '@wordpress/block-editor';
+import { InspectorControls, BlockControls, AlignmentToolbar, } from '@wordpress/block-editor';
```

Comment:
Let's keep this either as
```suggestion
import { InspectorControls, BlockControls, AlignmentToolbar } from '@wordpress/block-editor';
```
OR
```suggestion
import {
	InspectorControls,
	BlockControls,
	AlignmentToolbar,
} from '@wordpress/block-editor';
```

Reply:
Made changes.